src/trapfunc.cpp

```diff
@@ -822,17 +822,23 @@ void trapfunc::lava(Creature *c, int x, int y)
 }
 
 // STUB
-void trapfunc::portal(Creature */*c*/, int /*x*/, int /*y*/)
-{
+void trapfunc::portal(
+    Creature * //c
+  , int //x
+  , int //y
+) {
     // TODO: make this do something?
 }
 
-void trapfunc::sinkhole(Creature *c, int /*x*/, int /*y*/)
-{
-    if (c != &g->u) {
+void trapfunc::sinkhole(
+    Creature * //c
+  , int //x
+  , int //y
+) {
+    //if (c != &g->u) {
```

Comment:
generally speaking, never comment out code.  If it shouldn't be there, delete it.